test/test_re2.fx

```diff
@@ -0,0 +1,82 @@
+from UTest import *
+import Re2
+
+TEST("Re2 simple", fun() 
```

Comment:
to bring all the tests to the uniform style, could you please un-indent all the test bodies
```
    {
<====    ...
    }
```

and also replace spaces with dots in the test names?

Reply:
Done.